### DISCUSSION TITLE
"PII Disclosure" issue impact our root domain being marked as Deceptive Content

### DISCUSSION THREAD

**Discussion by saggafarsyad** [dani-garcia/vaultwarden#3501] [Q&A] [answered]
### Subject of the issue
In url <BASE_URL>/app/main.3a27378a743dd4ad9f70.js contains Credit Card Number "4242424242424242" which contains Personally Identifiable Information, such as CC number, SSN and similar sensitive data.

We suspect that this alert is a false alarm because the hardcoded text is intended as example or placeholder. But this PII Disclosure issue impact our root domain being marked as Deceptive content by Google and all our websites within subdomain were marked as Dangerous.

#### Solution
We tried replace "4242424242424242" to "42424242xxxxxxxx" on compiled file `/web-vault/app/main.3a27378a743dd4ad9f70.js`, re-deploy and re-scan OWASP ZAP, then the issue is resolved.

### Deployment environment

* vaultwarden version: 1.28.1

* Install method: Docker image

* Clients used: Web Vault, Chrome Extensions 

* Reverse proxy and version: NGINX v1.24.0 

* MySQL/MariaDB or PostgreSQL version: N/A 

* Other relevant details: Web Vault Version 2023.3.0

### Steps to reproduce
* Scan with OWASP ZAP

### Expected behaviour

* No High Risk Alerts

### Actual behaviour

* OWASP ZAP catch dummy Credit Card Number "4242424242424242" as PII Disclosure issue and marked as High Risk alert by OWASP ZAP scanner

### Troubleshooting data

* Related code
  <img width="1644" alt="Screenshot 2023-05-02 at 11 42 43" src="https://user-images.githubusercontent.com/7218502/235574801-95494019-b219-4497-b883-79be9f8aae3f.png">
* Related CWE: https://cwe.mitre.org/data/definitions/359.html
* OWASP ZAP Plugin: https://www.zaproxy.org/docs/alerts/10062/

**Comment by BlackDex** [comment]
This is not something we can fix for one. This code is part of the Bitwarden Clients repo https://github.com/search?q=repo%3Abitwarden%2Fclients%2042424242&type=code .

Second, that is a known test credit-card number. So i suggest to create a ticket at the detection library and report to let it exclude this and maybe other well knows test numbers.

**Reply by BlackDex** [reply, marked as answer]
Also, you mention that Google has marked your site as Deceptive.
That is a known issue if you are using the name `bitwarden` as a sub-domain for example.
Either change it to `vaultwarden` or something else not matching anything with `bitwarden`.

This is not something we can adjust.

**Reply by saggafarsyad** [reply]
Ah I see, that would make sense. Thank you. In the meantime I will migrate the subdomain to vaultwarden
